lib/assets/javascripts/cartodb/table/tableview.js

```diff
@@ -785,6 +785,11 @@
         this.add_related_model(this.geocoder);
       },
 
+      _changeTitle: function() {
+        this.$('.table_title').remove();
+        this.$el.prepend(cdb.templates.getTemplate('table/views/table_title')({'tablename': this.model.get('name')}));
```

Comment:
Might consider storing the thing created here on `this` so it can be referenced by `this.{its variable name}.$el` above for removal, instead of using a selector.